autoload/neomake.vim

```diff
@@ -845,27 +885,31 @@ function! neomake#ProcessPendingOutput() abort
         endif
     endif
     call neomake#highlights#ShowHighlights()
+
+    call s:process_queued_events()
```

Comment:
Should/can be done also when job output processing is not possible currently.. => move.